morphos/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: morphos_web_1
+      APP_PORT: 8080
+
+  web:
+    image: ghcr.io/danvergara/morphos-server:0.2.0
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - 8080:8080
+    volumes:
+      - ${APP_DATA_DIR}/tmp:/tmp
```

Comment:
Do you need to persist the `tmp` directory on the host machine between app restarts? If not you can just remove this entire directive.

If you do need the files in `tmp` to persist across app restarts then leave this as-is, but we'll beed to commit this directory to the app repo so that it gets installed with correct permissions on a users Umbrel. If we don't do this then Docker will create the directories with root permissions. What you can do is create a `data` directory in the morphos app directory, and then inside the `data` directory create a `tmp` directory with a `.gitkeep` file in it to make sure it is committed even though it is an empty directory.

Take a look at Gitea as an example: https://github.com/getumbrel/umbrel-apps/tree/master/gitea